### DISCUSSION TITLE
Extension settings aren't stored

### DISCUSSION THREAD

**Discussion by Joergw666** [Ulauncher/Ulauncher#1301] [General]
In a nearly fresh install of Ubuntu 22.04 settings of extensions aren't stored, whenever I change a setting, close the dialogue and reopen it, the old settings are there. I checked the write permissions for ~/.config/ulauncher, but they are there, inside the directory are a  couple of small files corresponding with the extensions installed. Any other place I should check? Any other suggestions?

**Comment by Joergw666** [comment]
I found the culprit by running ulauncher -v from the terminal: 
"Cannot update preferences. The extension controller is not registered. Try relaunching or restarting Ulauncher, and check the extension readme for external dependencies you may have to install manually."
How can I do this?

**Comment by friday** [comment]
It's because you didn't follow the installation instruction in the extension page most likely. The inconsistency in Ulauncher's handling of this should be fixed in https://github.com/Ulauncher/Ulauncher/releases/tag/5.15.5
